site/docs/community.md

```diff
@@ -36,13 +36,14 @@ Issues are tracked in GitHub:
 
 ### Contributing
 
-Iceberg uses Apache's GitHub integration. The code is available at https://github.com/apache/incubator-iceberg
+Iceberg uses Apache's GitHub integration. The code is available at [https://github.com/apache/incubator-iceberg].
 
 The Iceberg community prefers to receive contributions as [Github pull requests][github-pr-docs].
 
 * [View open pull requests][iceberg-prs]
 * [Learn about pull requests][github-pr-docs]
 
+[https://github.com/apache/incubator-iceberg]: https://github.com/apache/incubator-iceberg
```

Comment:
I wonder why these links are not showing in the markdown preview. Do you know why?

Reply:
Are these proper markdown? I've always used `[text](url)` or `[text][url-ref]`, not `[url]`.

Reply:
@rdsr Somehow mkdocs does not turn this http url is into a link automatically.

Reply:
@rblue This a reference link. See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links.

Reply:
Using the link text itself for a reference seems odd to me. I think we should add a markdown/mkdocs plugin to linkify URLs instead. There should be a plugin for this.

Reply:
It turned out we can just use Markdown's automatic links. Updated the PR.